src/utils/id-generator.js

```diff
@@ -22,3 +22,15 @@ export function sequentialIdGenerator() {
   let counter = 0;
   return () => String(++counter);
 }
+
+/**
+ * Returns a function that generates a random id in string format.  The random
+ * id is an [a-zA-Z] character followed by an integer from 0-99,999.
+ * @return {function():string}
+ */
+export function randomIdGenerator() {
+  return () =>
+    `${'abcdefghijklmnopqrstuvxwyzABCDEFGHIJKLMNOPQRSTUVWXYZ'.charAt(
```

Comment:
IMO, adding a non-numeric first char is the job for the caller, not this function. This is b/c for some use cases all-numeric IDs are totally fine. It just happens to be that for DOM IDs we want to have some restrictions. And for a caller the first char does not need to be random - it just needs any letter. E.g. the caller can just do `x${randomIdGenerator()()}`. And I think in your case, you already have a prefix and this is just a suffix, so there's no need to worry about the contents at all.

Reply:
Makes sense to me, updated!